addOns/authhelper/authhelper.gradle.kts

```diff
@@ -69,6 +78,16 @@ dependencies {
     zapAddOn("selenium")
     zapAddOn("spiderAjax")
     zapAddOn("client")
+    zapAddOn("zest")
+
+    implementation("org.zaproxy:zest:0.22.0") {
```

Comment:
This should be removed, this is no longer needed as the Zest library is now exposed as part of the API of the Zest add-on. (It's also outdated.)